Diplomatic/App.xaml

```diff
@@ -15,6 +16,31 @@
             <Setter Property="WidthRequest" Value="75" />
         </Style>
 
+        <Style TargetType="Label">
+            <Setter Property="Opacity" Value="0.5"/>
+            <Style.Triggers>
+                <Trigger TargetType="Label"
+                         Property="IsVisible" Value="True">
+                    <Trigger.EnterActions>
+                        <local:OpacityTriggerAction StartsFrom="0"/>
+                    </Trigger.EnterActions>
+                    <Trigger.ExitActions>
+                        <local:OpacityTriggerAction StartsFrom="1"/>
```

Comment:
I don't think this last exit-trigger is necessary. The label will be hidden before the trigger action starts, so the animation won't be visible.